mantidproject/mantid
# Fix sip generation rules for MantidPlot

**Description of work.**

#23594 broke the generation of the sip `.h`/`.cpp` files from the `.sip` files resulting in a broken [clean build](http://builds.mantidproject.org/view/Master%20Pipeline/job/master_clean-rhel7/809/console). This fixes the issue while keeping the fix behaviour that #23594 was attempting to fix.

**To test:**

A clean build should build MantidPlot successfully. Alternatively,

```
cd build
find MantidPlot -name '*sip*' -delete
ninja
```
should not work without this fix merged.

*There is no associated issue.*

*This does not require release notes* because **it fixes the build!**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
